src-cpp/rdkafkacpp.h

```diff
@@ -548,6 +548,13 @@ class Consumer : public virtual Handle {
                                 int timeout_ms,
                                 ConsumeCb *consume_cb,
                                 void *opaque) = 0;
+
+  /**
+   * Converts an offset into an offset from the tail of a topic.
```

Comment:
should probably say "into the logical offset from the tail of a topic".
Otherwise people might think that the returned value is an absolute offset.